Sources/SymbolKit/Mixin.swift

```diff
@@ -21,4 +25,66 @@ public protocol Mixin: Codable {
      > Important: With respect to deserialization, this framework assumes `mixinKey`s between instances of `SymbolMixin` are unique.
      */
     static var mixinKey: String { get }
+    
+    /// Defines the behavior when an error occurs while decoding this type of ``Mixin``.
+    ///
+    /// Next to logging warnings, the function allows for either re-throwing the error,
+    /// skipping the errornous entry, or providing a default value.
+    static func onDecodingError(_ error: Error) throws -> Mixin?
```

Comment:
This doesn't seem like it belongs on the Mixin model type. It's purely related to the coding process of the model.

Reply:
I could also implement this using the `userInfo` on `Encoder`/`Decoder` meaning that one would have to also `register` this behavior for custom Mixins.  Does that sound good?

Reply:
Yes, sure. I'll try it that way!

Reply:
Resolved in b93d460ebfdd40a60cd5304ff45a4dfc8fa45596.